### ISSUE TITLE
Variables and Operators: <Missing mention for naming variables>

### ISSUE THREAD

**Issue by DefalterXD** [TheOdinProject/curriculum#28811]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

In the lesson "Variables and operators", it does not mention anything about variable naming. It appears only after "knowledge check" with a supposed article about it. Maybe attach a link to that article in the section about variables?

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-variables-and-operators

**Comment by CouchofTomato** [comment]
@TheOdinProject/javascript Anyone have a view on this?

**Comment by MaoShizhong** [comment]
Good spot! I'm surprised that javascript.info variables lesson isn't actually in the main lesson content - I thought it was. I agree that given it's a sensible knowledge check question, it ought to link to something from the lesson itself, be that a section or something we link to.

I think a brief assignment step in the Assignment section "Go through the following articles to deepen your knowledge" bit would suit the variables link. Just a brief first step like

> 1. Read up on [variables in JavaScript](https://javascript.info/variables) from JavaScript.info.

Since you did not check the assignment box, I'll open this up for contribution.

---

## Acceptance criteria

- [ ] The Assignment section, just after the "Go through the following articles to deepen your knowledge" bit, has a first step added that links to the variables lesson from javascript.info, similar to the above.

Please comment below if you'd like to be assigned to this issue. Please do not open a PR unless you have been assigned by a maintainer.

**Comment by DefalterXD** [comment]
I'm sorry, I overlooked the assignment box to mark it. Can I too be assigned to this issue?

**Comment by MaoShizhong** [comment]
Of course you can
